### PR TITLE
141 cross browser

### PR DESCRIPTION
**Pull request by allthesignals**
🤔

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,15 @@ Make use of the many generators for code, try `ember help generate` for more det
 * `ember test`
 * `ember test --server`
 
+### Browserstack testing
```
Comment (andycochran) [inline README.md]: I can't get the tests running locally. It's not getting the keys from my `.env` file.

Comment (allthesignals) [reply README.md]: The tests should run fine locally `ember test`.

Comment (allthesignals) [reply README.md]: I'm gonna see about getting it to pick up the `.env` file properly

**Comment by chriswhong** [comment]
Closes #141

**Pull request merged**